packages/file_selector/file_selector/test/file_selector_test.dart

```diff
@@ -374,12 +448,35 @@ class FakeFileSelector extends Fake
     String? initialDirectory,
     String? suggestedName,
     String? confirmButtonText,
+  }) async {
+    return (await getSaveLocation(
+      acceptedTypeGroups: acceptedTypeGroups,
+      options: SaveDialogOptions(
+        initialDirectory: initialDirectory,
+        suggestedName: suggestedName,
+        confirmButtonText: confirmButtonText,
+      ),
+    ))
+        ?.path;
```

Comment:
Was this intentional (or produced with dartfmt)?

Reply:
`dartfmt` :( But @ditman's suggestion fixed it.